src/wallet/wallet.cpp

```diff
@@ -2495,13 +2495,16 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const int nConfMin
     }
 }
 
-bool CWallet::SelectCoins(const std::vector<COutput>& vAvailableCoins, const CAmount& nTargetValue, std::set<CInputCoin>& setCoinsRet, CAmount& nValueRet, CAmount not_input_fees, const CFeeRate effective_fee, const CCoinControl& coin_control, bool use_bnb, int change_output_size, int change_spend_size) const
+bool CWallet::SelectCoins(const std::vector<COutput>& vAvailableCoins, const CAmount& nTargetValue, std::set<CInputCoin>& setCoinsRet, CAmount& nValueRet, CAmount not_input_fees, const CFeeRate effective_fee, const CCoinControl& coin_control, bool& use_bnb, int change_output_size, int change_spend_size) const
 {
     std::vector<COutput> vCoins(vAvailableCoins);
 
     // coin control -> return all selected outputs (we want all selected to go into the transaction for sure)
     if (coin_control.HasSelected() && !coin_control.fAllowOtherInputs)
     {
+        // For now, don't use BnB if preset inputs are selected. TODO: Enable this later
```

Comment:
this is not a TODO, if there is no coin selection to be had, you cannot do coin selection.

Reply:
Done

Reply:
Huh?  
There is still coin selection if you have preset inputs.  Your preset inputs may not be sufficient, it just means use at least these coins.

Reply:
@morcos in this case the user has selected to not allow any other inputs?

Reply:
oops